git_machete/tests/functional/test_machete.py

```diff
@@ -539,7 +539,7 @@ def test_traverse_no_push(self) -> None:
     def test_traverse_no_push_override(self) -> None:
         self.setup_discover_standard_tree()
 
-        self.launch_command("traverse", "-Wy", "--no-push", "--push")
+        self.launch_command("t", "-Wy", "--no-push", "--push")
```

Comment:
Pls also add `--start-from=here` while you're here (as a regression test for https://github.com/VirtusLab/git-machete/pull/349)